modules/lead/toolchain/rode.tf

```diff
@@ -0,0 +1,63 @@
+module "ca-issuer" {
+  source = "../../common/ca-issuer"
+
+  enabled          = var.enable_rode
+  name             = "rode"
+  namespace        = var.namespace
+  common_name      = var.root_zone_name
+  cert-manager-crd = var.crd_waiter
+}
+
+module "grafeas_certificate" {
+  source = "../../common/certificates"
+
+  enabled         = var.enable_rode
+  name            = "grafeas-cert"
+  namespace       = var.namespace
+  domain          = "grafeas-server"
+  issuer_name     = module.ca-issuer.name
+  certificate_crd = var.crd_waiter
+  wait_for_cert   = true
+}
+
+module "rode_certificate" {
```

Comment:
I'm a bit confused by why this is needed.  the rode chart doesn't currently support passing your own certificates to it.  all you can do is provide a name for the secret that the rode chart creates for you. (https://github.com/liatrio/rode/blob/master/helm-chart/rode/templates/secrets.yaml#L29).

to be clear, I think rode definitely should support passing your own certificates (and potentially having that be the _only_ option), but since that's not how the chart works now, maybe this should be removed.

Reply:
Ah I see that now, that was just a failure of understanding the functionality on my part. I've made updated changes to pass a secret name for both Rode and Grafeas and removed the certificate creation.